includes/admin/views/html-send-return-email-form.php

```diff
@@ -26,7 +26,7 @@
 
 ?>
     <form
-        method="post" class="page-form wcmp__bulk-options wcmp__return-dialog" action="<?php echo $target_url; ?>">
+        method="post" class="page-form wcmp__bulk-options wcmp__return-dialog" action="<?php echo esc_url($target_url); ?>">
```

Comment:
Single quotes

Reply:
Dit is html, daar 'horen' de quotes double te zijn.